changelogs/fragments/v10.0.0_bump_ansible_core.yaml

```diff
@@ -0,0 +1,4 @@
+---
+major_changes:
```

Comment:
Can we use the same wording that we did before - https://github.com/ansible-collections/cisco.iosxr/pull/461/files#diff-badba4dd47cd1a24e2765eb5c5f420a529872e06a0ce09251c35ea2cb5b4ebd9R691-R699? Except for the "last known version compatible with...." part.